Mathlib.lean

```diff
@@ -5296,6 +5296,7 @@ import Mathlib.Tactic.Says
 import Mathlib.Tactic.ScopedNS
 import Mathlib.Tactic.Set
 import Mathlib.Tactic.SetLike
+import Mathlib.Tactic.Simp.FinsetInterval
```

Comment:
Suggest naming `Mathlib.Tactic.Simproc.FinsetInterval`. Clearer, and more consistent with Lean.